survey-sdk/src/main/java/co/nimblehq/survey/sdk/SurveyApi.kt

```diff
@@ -50,6 +52,11 @@ class SurveyApi private constructor() : NetworkBuilder() {
         return this
     }
 
+    fun addJsonApiClasses(vararg classes: Class<out Resource>): SurveyApi {
+        MoshiBuilderProvider.addJsonApiClasses(*classes)
+        return this
+    }
```

Comment:
this method should be marked as TODO too, same to `MoshiBuilderProvider.addJsonApiClasses`

Reply:
@luongvo Agree